integrations/ultralytics-yolov3/tutorials/yolov3_sparse_transfer_learning.md

```diff
@@ -137,6 +137,10 @@ The recipes are specific to the sparsification type, so the training command wil
         ```bash
         python train.py --data voc.yaml --cfg ../models/yolov3-spp.lrelu.yaml --weights DOWNLOADED_PATH --hyp data/hyp.finetune.yaml --recipe ../recipes/yolov3-spp.transfer_learn_pruned_quantized.md
         ```
+     Note, running quantization aware training (QAT) will consume up to three times the memory as FP32 training. 
+     It is recommended to use this recipe with a batch size that fits in memory for FP32 as normal at the start, though.
```

Comment:
why not FP16 if the script will use that by default?

Reply:
yep, good catch, that should say fp16